app/data/jsonSchemaForm/projectEmissionIntensitySchema.ts

```diff
@@ -0,0 +1,99 @@
+export const emissionIntensityReportSchema = {
+  $schema: "http://json-schema.org/draft-07/schema",
+  type: "object",
+  title: "Project TEIMP reporting",
+  required: [
+    "measurementPeriodStartDate",
+    "measurementPeriodEndDate",
+    "emissionFunctionalUnit",
+    "baselineEmissionIntensity",
+    "targetEmissionIntensity",
+  ],
+  properties: {
+    measurementPeriodStartDate: {
+      title: "Measurement period start date",
+      type: "string",
+    },
+    measurementPeriodEndDate: {
+      title: "Measurement period end date",
+      type: "string",
+    },
+    emissionFunctionalUnit: {
+      title: "Functional Unit",
+      type: "string",
+      default: undefined,
+    },
+    baselineEmissionIntensity: {
+      title: "Base Line Emission Intensity (BEI)",
+      type: "number",
+    },
+    targetEmissionIntensity: {
+      title: "Target Emission Intensity (TEI)",
+      type: "number",
+    },
+    postProjectEmissionIntensity: {
+      title: "Post Project Emission Intensity",
+      type: "number",
+    },
+    totalLifetimeEmissionReduction: {
+      title: "Total lifetime emissions reductions",
+      type: "number",
+    },
+  },
+};
+
+export const emissionIntensityReportingRequirementSchema = {
```

Comment:
Should each schema get its own file? Or, this one (and the ui schema below) are quite similar to the existing `reportingRequirement` one, maybe that could be reused?